RPM/sqlite-devel/rpmbuild/SPECS/sqlite-devel.spec

```diff
@@ -19,29 +19,32 @@ for sqlite. If you like to develop programs using sqlite, you will need
 to install sqlite-devel.
 
 %install
-mkdir %{buildroot}/bin
-mkdir -p %{buildroot}/usr/include
-mkdir -p %{buildroot}/lib64/pkgconfig
-cp /bin/sqlite3 %{buildroot}/bin/
-cp /usr/include/sqlite3.h %{buildroot}/usr/include/
-cp /usr/include/sqlite3ext.h %{buildroot}/usr/include/
-cp /lib64/libsqlite3.a %{buildroot}/lib64/
-cp /lib64/libsqlite3.la %{buildroot}/lib64/
-ln -s /lib64/libsqlite3.so.0.8.6 %{buildroot}/lib64/libsqlite3.so
-ln -s /lib64/libsqlite3.so.0.8.6 %{buildroot}/lib64/libsqlite3.so.0
-cp /lib64/libsqlite3.so.0.8.6 %{buildroot}/lib64/
-cp /lib64/pkgconfig/sqlite3.pc %{buildroot}/lib64/pkgconfig/
+mkdir -p %{buildroot}/opt/awn/bin
+mkdir -p %{buildroot}/opt/awn/usr/include
+mkdir -p %{buildroot}/opt/awn/lib64/pkgconfig
+mkdir -p %{buildroot}/etc/ld.so.conf.d
+cp /bin/sqlite3 %{buildroot}/opt/awn/bin/
+cp /usr/include/sqlite3.h %{buildroot}/opt/awn/usr/include/
+cp /usr/include/sqlite3ext.h %{buildroot}/opt/awn/usr/include/
+cp /lib64/libsqlite3.a %{buildroot}/opt/awn/lib64/
+cp /lib64/libsqlite3.la %{buildroot}/opt/awn/lib64/
+ln -s /lib64/libsqlite3.so.0.8.6 %{buildroot}/opt/awn/lib64/libsqlite3.so
+ln -s /lib64/libsqlite3.so.0.8.6 %{buildroot}/opt/awn/lib64/libsqlite3.so.0
```

Comment:
To reiterate my comment in the gvm-libs PR, I am curious about the outcome of the soft link here.